nomenclature/definitions/variable/fuel_types.yaml

```diff
@@ -0,0 +1,80 @@
+# This list of fuels will replace any occurrence of `<Fuel>`
+# in a variable-definition yaml file
+
+# All fuels that have the attribute "ccs: True" have two sub-categories
+# "w/ CCS": with a CO2 capture component
+# "w/o CCS": with freely vented CO2 emissions
+
+<Fuel>:
+  Biomass:
+    description: Purpose-grown biomass, crop residues, forest industry waste,
+                 municipal solid waste
+    ccs: True
+
+  Coal:
+    description: Coal (including hard coal and lignite)
+    ccs: True
+
+  Fossil:
+    description: All fossil fuels (crude oil, coal, natural gas)
+    ccs: True
+
+  Gas:
+    description: Natural gas (including methane from biomass or hydrogenation)
+    ccs: True
+
+  Gas|Fossil:
+    description: Natural gas (excluding methane from biomass or hydrogenation)
+    ccs: True
+
+  Gas|Biomethane:
+    description: Biomethane (upgraded biogas)
+    ccs: True
+
+  Gas|Synthetic Methane:
+    description: Synthetic methane (methanation of hydrogen)
+    ccs: True
+
+  Oil:
+    description: Refined liquid oil products
+    ccs: True
+
+  Nuclear:
+    description: Nuclear energy
+
+  Hydrogen:
+    description: Hydrogen
+
+  Non-Biomass Renewables:
+    description: Hydro, wind, solar, geothermal, ocean, and other renewable
+                 sources excluding bioenergy
+
+  Hydro:
+    description: Hydropower
+
```

Comment:
Hydro|Run of River:
     description: Hydropower without a large dam and reservoir. It only uses the energy from flowing water to generate electricity.

Hydro|Pumped Storage:
     description: This hydropower plant has two water reservoirs at different elevations: one at the upper level and one at the lower level. It can generate power (discharge) as water moves down through a turbine; however, it draws power as it pumps water (recharge) to the reservoir at the upper level.

Reply:
and also Hydro|Reservoir :-)

Reply:
Yeah, I agree. Thanks @sandrinecharousset .